src/wallet/rpcwallet.cpp

```diff
@@ -223,6 +223,9 @@ static void SetFeeEstimateMode(const CWallet* pwallet, CCoinControl& cc, const U
         }
 
         cc.m_feerate = CFeeRate(fee_rate);
+        if (*cc.m_feerate <= CFeeRate(0)) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid fee_rate %s (must be greater than 0)", cc.m_feerate->ToString()));
+        }
```

Comment:
It could maybe just be removed. The tests I added in #20220 for the other 5 RPCs check that the code detects this and raises with `"Fee rate (0.00000000 BTC/kB) is lower than the minimum fee rate setting (0.00001000 BTC/kB)`

Reply:
IMO these are fundamentally different error types, and should probably be handled differently.